README.md

```diff
@@ -9,9 +9,9 @@
 
```

Comment:
Would you mind this change to also move the tags form the headline to the other two for Travis and GoDoc?